packages/fxa-settings/src/components/Settings/SubRow/index.tsx

```diff
@@ -265,8 +265,7 @@ export const BackupPhoneSubRow = ({
 
   const linkExternalProps = !hasPhoneNumber
     ? {
-        // TODO add a link to the knowledge base article once it is available
-        href: '',
+        href: 'https://support.mozilla.org/en-US/kb/secure-firefox-account-two-step-authentication',
```

Comment:
This link should point to the new article about value and risks related to 2FA and recovery methods:

```suggestion
        href: 'https://support.mozilla.org/kb/secure-mozilla-account-two-step-authentication',
```
The other link goes to setup instructions for 2FA. Also noting that we shouldn't force the locale when linking to SUMO (= we should remove `en-US` from the links.